projects/crypto-currency/iluminados/app/src/components/Influencer/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from 'react'
+import './style.css'
+
+function Influencer() {
+
+  const [list, setList] = useState([])
+  const API = process.env.REACT_APP_API_URL
+
+  useEffect(() => {
+    fetch(
+      `${API}influencers`,
+      { method: 'get' }
+    )
+      .then(async response => {
+        const { data } = await response.json()
+        setList([data])
+      })
+      .catch(error => console.log(error))
+  },
+  )
+
+  const renderinfluencer = () => {
```

Comment:
`renderinfluencer ` -> `renderInfluencer `
```suggestion
  const renderInfluencer = () => {
```